news/xonfig-wizard.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+
+* Foreign shell xonfig wizard empty shell name entry
```

Comment:
Hey @laerus -- would you mind adding a note here that an empty foreign shell entry will now default to `bash`?